src/Export/Data/PriceCurrencyFields.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Omikron\FactFinder\Shopware6\Export\Data;
+
+use Omikron\FactFinder\Shopware6\Config\ExportSettings;
+use Omikron\FactFinder\Shopware6\Export\Field\PriceCurrency;
+use Shopware\Core\Framework\Api\Context\SystemSource;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\System\Currency\CurrencyEntity;
+
+class PriceCurrencyFields
+{
+    /** @var EntityRepositoryInterface */
+    private $currencyRepository;
+
+    /** @var ExportSettings */
+    private $exportFilters;
+
+    /** @var array */
+    private $currencyFields = [];
+
+    /**
+     * @param EntityRepositoryInterface $currencyRepository
```

Comment:
Docblock is not necessary i think, besides of that it does not match the real signature of the constructor